frontend/class-twitter.php

```diff
@@ -158,6 +158,9 @@ protected function description() {
 		if ( is_singular() ) {
 			$meta_desc = $this->single_description();
 		}
+		elseif ( is_home() && ! is_front_page() ) {
```

Comment:
Ok, so no I'm starting to think `is_posts_page` could be a function of `WPSEO_Utils`